webapi/src/main/scala/org/knora/webapi/slice/infrastructure/api/ManagementEndpoints.scala

```diff
@@ -85,6 +86,11 @@ final case class ManagementEndpoints(baseEndpoints: BaseEndpoints) {
     .out(jsonBody[HealthResponse])
     .out(statusCode)
 
+  private[infrastructure] val postStartCompaction = baseEndpoints.publicEndpoint.post
```

Comment:
question (blocking): I overlooked this at first... but if you use `publicEndpoint` does that mean that this can be triggered without authentication?  
This should only be possible for system admins, I suppose.

Reply:
ClassTooLarge problem encountered.

I have solved by removing the top layer all the dependencies from the dep. graph that, the ones that were "top" dependencies (all the actor stuff is final, for example) are now extracted. This method should be reappliable, but though it's not quite trivial.

Anyhow, the PR is tested, hoping for a green build now.